lib/Elastica/Suggest/AbstractSuggest.php

```diff
@@ -16,9 +17,9 @@
     protected $_name;
 
     /**
-     * @var string the text for this suggestion
+     * @var string the prefix for this suggestion
      */
-    protected $_text;
+    protected $_prefix;
```

Comment:
@ruflin Yes you'r right both params still exists, in other way https://github.com/ruflin/Elastica/pull/1213 this one wasn't successful, and idea to remove variable sounds good to me, but i think we should keep only only "prefix" param to avoid misunderstanding!